frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -34,20 +33,22 @@ import {
   VolumeSnapshotModel,
   VolumeSnapshotClassModel,
 } from '@console/internal/models';
-import { getName, getNamespace, Status, isCephProvisioner } from '@console/shared';
+import { getName, getNamespace, Status, isCephProvisioner, getAnnotations } from '@console/shared';
 import { StorageClassDropdown } from '@console/internal/components/utils/storage-class-dropdown';
-import {
-  dropdownUnits,
-  cephRBDProvisionerSuffix,
-} from '@console/internal/components/storage/shared';
+import { dropdownUnits } from '@console/internal/components/storage/shared';
 import { useK8sGet } from '@console/internal/components/utils/k8s-get-hook';
 import { AccessModeSelector } from '../../access-modes/access-mode';
 import './restore-pvc-modal.scss';
+import { RadioInput } from '@console/internal/components/radio';
 
 const RestorePVCModal = withHandlePromise<RestorePVCModalProps>(
   ({ close, cancel, resource, errorMessage, inProgress, handlePromise }) => {
     const { t } = useTranslation();
+    const snapshotPVCStorageClassAnnotation = 'snapshot.storage.kubernetes.io/pvc-storage-class';
+    const snapshotPVCAccessModeAnnotation = 'snapshot.storage.kubernetes.io/pvc-access-modes';
+    const snapshotPVCVolumeModeAnnotation = 'snapshot.storage.kubernetes.io/pvc-volume-mode';
```

Comment:
Define this variable in a shared folder and use them.

Reply:
done